test/extended/networking.sh

```diff
@@ -189,7 +214,7 @@ ${OPENSHIFT_INSTANCE_PREFIX:-openshift}"
     fi
     ;;
   *)
-    if [ "${CONFIG_ROOT}" != "" ]; then
+    if [ -n "${CONFIG_ROOT}" ]; then
```

Comment:
Also, to guard against `$CONFIG_ROOT` being unset, we can default it: `if [[ -n "${CONFIG_ROOT:-}" ]]; then ...`